Appium/wireless_connect.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ ! -z "$ANDROID_DEVICES" ]; then
+    connected_devices=$(adb devices)
+    IFS=',' read -r -a array <<< "$ANDROID_DEVICES"
+    for i in "${!array[@]}"
+    do
+        array_device=$(echo ${array[$i]} | tr -d " ")
+        #string contains check
+        if [[ ${connected_devices} != *${array_device}* ]]; then
+            echo "Connecting to: ${array_device}"
+            adb connect ${array_device}
```

Comment:
should we check the exit code here to make sure the connection was really successful?

Reply:
No, we don't care. Anyway script will be run again after specified delay.
It is the main idea of this PR. Script will try to reconnect to remote device in any case